mu4e/mu4e-message.el

```diff
@@ -181,6 +182,17 @@ representation."
       ;; otherwise, use text
       (t nil))))
 
+(defun mu4e~want-format-flowed (msg)
```

Comment:
Hmm, I think `mu4e~message-body-has-content-type-param` would be better, with parameters `msg' and some string; furthermore, seems that it's a property of a specific mime-part, not necessarily of the message as a whole, or?